packages/patternfly-4/react-core/src/components/Dropdown/DropdownToggleCheckbox.tsx

```diff
@@ -16,6 +16,8 @@ export interface DropdownToggleCheckboxProps extends Omit<React.HTMLProps<HTMLIn
   checked?: boolean | null;
   /** A callback for when the checkbox selection changes */
   onChange?(checked: boolean, event: React.FormEvent<HTMLInputElement>): void;
+  /** */
```

Comment:
Add comment: `Element to be rendered inside the <span>`

Reply:
Yeah, good point. Missed that.